lib/core/src/model.rs

```diff
@@ -1926,8 +1933,10 @@ pub struct PrepareLnUrlPayResponse {
     pub destination: SendDestination,
     /// The fees in satoshis to send the payment
     pub fees_sat: u64,
-    /// The [LnUrlPayRequestData] returned by [crate::input_parser::parse]
+    /// The [LnUrlPayRequestData] returned by [parse]
     pub data: LnUrlPayRequestData,
+    /// A BIP353 address, in case one was used in order to fetch the LNURL Pay request data.
+    pub bip353_address: Option<String>,
```

Comment:
I didn't dig too deep, but is there a reason we need the `bip353_address` here and in the SendDestination? 

I'm wondering if it's enough add `bip353_address` to the Bolt11 variant also and to have `prepare_lnurl_pay()` fill in the `bip353_address` in the SendDestination response from its call to `prepare_send_payment()`

Reply:
I didn't consider that before, but I like that it improves consistency. See [d004300](https://github.com/breez/breez-sdk-liquid/pull/718/commits/d00430006353a09348e8f7f79d8f34012d64ce6c).